app-header/test/app-header.html

```diff
@@ -119,9 +120,10 @@
           tearDown: sinon.spy(),
           run: sinon.spy()
         };
-        Polymer.AppLayout.scrollEffects = {
-          'test-effect': testEffect
-        };
+
+        effectCount++;
+        effectName = 'test-effect-' + effectCount;
```

Comment:
Two reasons: 1. `setup` is ran multiple times and now `registerEffect` checks that the effect hasn't been registered yet (to help developers troubleshooting their apps).  2. there isn't a way to unregister effects yet bc I didn't find a use case for that.